ROLES.md

```diff
@@ -204,8 +214,14 @@ If a maintainer becomes inactive in the project for an extended period of time,
 emeritus maintainer. Emeritus maintainers lose their ability to approve code contributions, but retain their voting rights
 for up to one year. After one year, emeritus maintainers revert back to being normal members. 
 
-A maintainer becomes an emeritus maintainer if the individual hasn't contributed a PR to the project in 3 months. An emeritus 
-maintainer can regain full maintainer status by submitting 5 PRs to the project.
+Contributing to the project can be done in a number of ways:
```

Comment:
Also up in the "Requirements" section we mention must remain an active participant, maybe we can reword that and point down to the later section describing how to contribute.

Alternatively we could just expand the requirements section a bit to provide more detail, since it already covers contributing code, doing reviews, and triaging issues. So we could just add more text up there.

Reply:
I combined the sections, seems cleaner that way.